### ISSUE TITLE
I can't get select2 dependent fields work

### ISSUE THREAD

**Issue by bhdrkcr** [codingjoe/django-select2#53]
I am trying to implement dependent select fields as shown in documents .

My models are shown here:
```

class VehicleBrand(BaseModel):
"""Brands for types of vehicles"""

name = models.CharField(_('Name'), max_length=50)

def __str__(self):
    return self.name


class VehicleModel(BaseModel):
"""vehicle model"""

brand = models.ForeignKey(
    'awesomeinventory.VehicleBrand',
    verbose_name=_('Brand of the vehicle'),
    on_delete=models.CASCADE,
    related_name='models',
)
name = models.CharField(_("Vehicle model name"), max_length=50)

def __str__(self):
    return self.name
```

My forms.py looks like:
```

class MarketTrackForm(forms.Form):
brand = forms.ModelChoiceField(
    queryset=VehicleBrand.objects.all(),
    to_field_name='name',
    label=u'Brand',
    widget=ModelSelect2Widget(
        model=VehicleBrand,
        search_fields=['name__icontains'],
        attrs={'style': 'width: 100%', 'data-minimum-input-length': 0},
    ),
)
model = forms.ModelChoiceField(
    queryset=VehicleModel.objects.all(),
    to_field_name='name',
    label=u'Model',
    widget=ModelSelect2Widget(
        model=VehicleModel,
        search_fields=['name__icontains'],
        dependent_fields={'brand': 'brand'},
        max_results=500,
        attrs={
            'style': 'width: 100%',
            'data-minimum-input-length': 0,
        },
    ),
)
```
my urls.py
```

urlpatterns += i18n_patterns(
...
path('select2/', include('django_select2.urls')),
...
)

```
and my settings looks like
```

...
    CACHES = {
    "default": {
        "BACKEND": "django_redis.cache.RedisCache",
        "LOCATION": "redis://127.0.0.1:6379/1",
        "OPTIONS": {
            "CLIENT_CLASS": "django_redis.client.DefaultClient",
        },
    },
    'select2': {
        "BACKEND": "django_redis.cache.RedisCache",
        "LOCATION": "redis://127.0.0.1:6379/2",
        "OPTIONS": {
            "CLIENT_CLASS": "django_redis.client.DefaultClient",
        },
    },
}

# Set the cache backend to select2
SELECT2_CACHE_BACKEND = 'select2'
...

```

When I remove dependent fields everything works. Also I tried to copy exact example shown in documentation (country and city models) and even that didn't work. when I add dependent field, it just starts to say no results found(after selecting dependent field), If I remove dependent field from form it works. So I thought it could be about settings or caching. But couldnt figure it out. Any tips would help, thank you.

---update---
When I remove

```

kwargs.update(
    {
        f"{model_field_name}__in": filter(
            None, self.request.GET.get(f"{form_field_name}[]", "").split(",")
        )
        for form_field_name, model_field_name in self.widget.dependent_fields.items()
    }
)
```
This code from select2.views.AutoResponseView it just started to work. I dont want to mess with it tho. How could I fix it without removing this pile of code and what even this pile does? anyone?

**Comment by bhdrkcr** [comment]
changing ModelSelect2Widget to ModelSelect2MultipleWidget solved the problem.

**Comment by codingjoe** [comment]
There also were too little bugs in the latest 7.7.0 release that was fixed in 7.7.1
